### PR TITLE
Fix/offer page varianted options title

### PR DESCRIPTION
**Pull request by StylianosGakis**
Screenshot_20220429-175848_Hedvig Dev

### CODE REVIEW

**Review by hugokallstrom — APPROVED** [review]
LGTM

**Comment by StylianosGakis** [comment]
Forgot to mention, just gonna wait till we have real translations before merge.

**Pull request merged**
